vhdbuilder/packer/windows/configure-windows-vhd.ps1

```diff
@@ -903,6 +903,9 @@ try
             Clear-TempFolder
             Log-VHDFreeSize
             Test-AzureExtensions
+
+            Write-Output "creating test.txt file"
+            Write-Output "this is a test file" > "c:\k\test.txt"
```

Comment:
Reminder to Tim: I have to remove this once I've tested the e2e tests are running against the right VHDs.